### DISCUSSION TITLE
Where is the source code for retype-cli exeuctables?

### DISCUSSION THREAD

**Discussion by axonasif** [retypeapp/retype#62] [Q&A] [answered]
Hi.
I'm a bit lost at finding the source code of retype-cli.
Or is it not available? Just wanted to know.
Thanks.

**Comment by geoffreymcgill** [comment, marked as answer]
The Retype app CLI (executable) is a closed source project and not publicly available.

**Reply by alinsimoc** [reply]
Are there any plans on making it open source? On the other side, is Retype going to stay free forever?

**Reply by geoffreymcgill** [reply]
Retype will remain closed-source, but there will _**always**_ be a freemium version.

The current limitation is 100 pages. More than 100 pages would require an upgrade to Retype Pro. So far, the VAST majority of projects we are seeing are well under 100 pages.

There will be other Pro only features that delivery a lot of extra value for projects that need that extra power. Almost everything will be available in the free version, although possibly with limitations, such as the 100 page limit.

Hope this helps.

**Reply by alinsimoc** [reply]
Thanks for the info!
